templates/server-acl-init-podsecuritypolicy.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if (or (and (ne (.Values.server.enabled | toString) "-") .Values.server.enabled) (and (eq (.Values.server.enabled | toString) "-") .Values.global.enabled)) }}
+{{- if .Values.global.bootstrapACLs }}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ template "consul.fullname" . }}-server-acl-init
+  labels:
+    app: {{ template "consul.name" . }}
+    chart: {{ template "consul.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+spec:
+  privileged: false
+  # Allow core volume types.
+  volumes:
+    - 'emptyDir'
+    - 'secret'
+    - 'downwardAPI'
```

Comment:
Secret is because every pod gets a service account token:
```
  volumes:
  - name: static-client-token-7ggpm
    secret:
      defaultMode: 420
      secretName: static-client-token-7ggpm
```

Downward API is because we use:
```
          env:
            - name: NAMESPACE
              valueFrom:
                fieldRef:
                  fieldPath: metadata.namespace
```

I think I can remove emptyDir.

Reply:
oh maybe I don't need downward either because that comes from an env var